.gitignore

```diff
@@ -18,6 +18,7 @@ transcript
 apps/elf/*
 apps/elf_txt/*
 apps/sv/*
+apps/asm/*
```

Comment:
What about the tests that you wrote in assembly? Don't you want then tk be committed?

Reply:
They are in apps/asm_verif, apps/asm now include just the asm files from the C files compilation.